content/shmem_alltoall_nb.tex

```diff
@@ -0,0 +1,133 @@
+\apisummary{
+  Exchanges a fixed amount of contiguous data blocks between all pairs
+  of \acp{PE} participating in the collective routine.
+}
+
+\begin{apidefinition}
+
+%% C11
+\begin{C11synopsis}
+int @\FuncDecl{shmem\_alltoall\_nb}@(shmem_team_t team, TYPE *dest, const TYPE
+*source, size_t nelems, shmem_req_h *request);
+\end{C11synopsis}
+where \TYPE{} is one of the standard \ac{RMA} types specified by Table \ref{stdrmatypes}.
+
+\begin{Csynopsis}
+\end{Csynopsis}
+\begin{CsynopsisCol}
+int @\FuncDecl{shmem\_\FuncParam{TYPENAME}\_alltoall\_nb}@(shmem_team_t team,
+TYPE *dest, const TYPE *source, size_t nelems, shmem_req_h *request);
+\end{CsynopsisCol}
+where \TYPE{} is one of the standard \ac{RMA} types and has a corresponding \TYPENAME{} specified by Table \ref{stdrmatypes}.
+
+\begin{CsynopsisCol}
+int @\FuncDecl{shmem\_alltoall32\_nb}@(shmem_team_t team, void *dest, const
+void *source, size_t nelems, shmem_req_h *request);
+
+int @\FuncDecl{shmem\_alltoall64\_nb}@(shmem_team_t team, void *dest, const
+void *source, size_t nelems, shmem_req_h *request);
+
+int @\FuncDecl{shmem\_alltoallmem\_nb}@(shmem_team_t team, void *dest, const
+void *source, size_t nelems, shmem_req_h *request);
+\end{CsynopsisCol}
+
+\begin{apiarguments}
+
+\apiargument{IN}{team}{A valid \openshmem team handle to a team.}%
+
+\apiargument{OUT}{dest}{Symmetric address of a data object large enough to receive
+    the combined total of \VAR{nelems} elements from each \ac{PE} in the
+    active set.
```

Comment:
...in the "team"?

If so, we have the same wording in blocking alltoall.  So this was inherited :)